docs/topics/crud-mission-database-interaction.adoc

```diff
@@ -1,5 +1,5 @@
 [[common-crud-database-interaction]]
-= Interacting with the CRUD API
+= Interacting with the Relational Database Backend API
```

Comment:
I'd personally be fine with calling all applications just "application" :-), but "CRUD Application" is perfectly fine. It's a good description of what the application does.